docs/nederlands-overzicht.md

```diff
@@ -7,6 +7,54 @@ Dit kan vormgegeven zijn met een Open Source Program Office of op een andere man
 
 Zie de [Wikipedia pagina Publieke sector](https://nl.wikipedia.org/wiki/Publieke_sector) voor een indicatie van de verschillende organisaties die hieronder vallen. Dit beslaat zowel overheid als semi-overheidsorganisaties.
 
+// [:material-web:](https://www.alliander.com/nl/open-source/)
+// [:material-github:](https://github.com/MinBZK)
+
+## Ministeries
+- Ministerie van Algemene Zaken
+- Ministerie van Binnenlandse Zaken en Koninkrijksrelaties (BZK)
+    - BZK 
+      [:material-github:](https://github.com/MinBZK)
+      [:material-web:]()
+    - ICTU
+      [:material-github:](https://github.com/ICTU)
+      [:material-mail:](mailto:info@ictu.nl)
+      [:material-web:](https://www.ictu.nl)
+    - Kadaster
```

Comment:
Hmm... Wij zijn een ZBO en leggen wel verantwoording af aan het ministerie ... maar we _zijn_ geen ministerie ... 🤔

Reply:
En we staan er nu twee keer op (zo actief zijn we nu ook weer niet, hoor 😜 )

Reply:
Ik heb nog meer uitgewerkt en daar zit wel onderscheid in. Wilde eerst dit deel even werkend hebben ;-)